src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,524 @@
+"""Table definitions for EIA AEO resources."""
+
+from typing import Any
+
+# 2024-04-24: workaround so we can *define* a bunch of schemas without
+# documenting them or expecting assets to exist
+_STAGING_RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_eiaaeo__yearly_projection_electric_sector_generation_by_technology": {
```

Comment:
since our prs are twins rn, there is much overlap. @zaneselvans zane over there [suggested using `projected`](https://github.com/catalyst-cooperative/pudl/pull/3570#discussion_r1578269762) instead of `projection`. not actually sure why? zane why? is it bc it is verb-y? i like the ring of projected better